datalab/datalab_session/tests/test_operations.py

```diff
@@ -169,9 +169,9 @@ def tearDown(self):
         return super().tearDown()
 
     @mock.patch('datalab.datalab_session.file_utils.tempfile.NamedTemporaryFile')
-    @mock.patch('datalab.datalab_session.file_utils.get_fits')
-    @mock.patch('datalab.datalab_session.data_operations.median.save_fits_and_thumbnails')
-    @mock.patch('datalab.datalab_session.data_operations.median.create_jpgs')
+    @mock.patch('datalab.datalab_session.data_operations.data_operation.get_fits')
+    @mock.patch('datalab.datalab_session.file_utils.save_fits_and_thumbnails')
```

Comment:
Something I had noticed, but glossed over before, is the amount of mocking we're having to do which points to the fact that the code isn't easily testable (components are tightly coupled, for instance) and means we should consider re-factoring a bit.

Reply:
Aside from this, for Test classes that will mock a whole set of things to test a data operation for instance, you can mock out the common things at the class level so you don't have to repeat them for each test in the class.